docs/release-notes/7.17.asciidoc

```diff
@@ -5,6 +5,44 @@
 [[release-notes-7.17.6]]
 === 7.17.6
 
+[discrete]
+[[known-issue-7.17.6]]
+==== Known issues
+* In some situations, {elastic-endpoint} might change to a non-running state on Windows endpoints and cannot be restarted. {agent} will appear `Unhealthy` when this happens (https://github.com/elastic/endpoint/issues/29[#29]).
++
+To determine whether {elastic-endpoint} has stopped running because of this issue, run the following PowerShell command as an administrator:
+
++
+[source,console]
+--------------------------------------------------
+PS C:\Users\user> Get-WinEvent Microsoft-Windows-CodeIntegrity/Operational | where Id -eq 3004 | where Message -match "elastic-endpoint.exe"
+
+
+   ProviderName: Microsoft-Windows-CodeIntegrity
+
+TimeCreated                      Id LevelDisplayName Message
+-----------                      -- ---------------- -------
+9/22/2022 10:47:35 AM          3004 Error            Windows is unable to verify the image integrity of the file \Device\HarddiskVolume3\Program Files\Elastic\Endpoint\elastic-endpo...
+9/19/2022 2:10:14 PM           3004 Error            Windows is unable to verify the image integrity of the file \Device\HarddiskVolume3\Program Files\Elastic\Endpoint\elastic-endpo...
+--------------------------------------------------
+
++
+
+If {elastic-endpoint} is not running, there are several workarounds you can take:
+
+** **Trigger an Endpoint reinstall by manually uninstalling {elastic-endpoint} on affected hosts**: Remove an invalid {elastic-endpoint} install by manually invoking the {elastic-endpoint} <<uninstall-endpoint,uninstall command>> on affected hosts. Once the uninstallation process has finished, use the command below to restart {agent}. Restarting {agent} automatically begins {elastic-endpoint}'s re-installation process:
++
+[source,console]
+--------------------------------------------------
+c:\Program Files\Elastic\Agent\elastic-agent.exe restart
+--------------------------------------------------
+
+** **Uninstall, then re-install the {endpoint-cloud-sec} integration on affected hosts**: Uninstalling and re-installing the {endpoint-cloud-sec} integration on affected hosts will also force the uninstallation and re-installation of {elastic-endpoint} on these hosts.
++
+NOTE: Uninstalling the {endpoint-cloud-sec} integration may put {agent}  in an `Unhealthy` state. This is temporary and the state will change to `Healthy` once the integration is re-installed.
```

Comment:
Alt suggestion since in the Endpoints list  UI, it's `agent status`. 
```suggestion
NOTE: Uninstalling the {endpoint-cloud-sec} integration may temporarily cause {agent}'s status to be  `Unhealthy`. The status will change to `Healthy` once the integration is re-installed.
```